<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="2">
     <widget class="QLabel" name="label_usb_f3_conn">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Checking Ethernet Port 2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_usb_r3_conn">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="19" column="0" colspan="3">
     <widget class="QProgressBar" name="progressBar">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>D/D Loading</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Checking USB Port 2 (FRONT 3.0)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Action</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_eth1">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_usb_f2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Checking USB Port 3 (REAR 3.0)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QLabel" name="label_mac">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_version">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Ver.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_usb_r3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Checking Ethernet Port 1</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_usb_f2_conn">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>MAC ADDRESS</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_eth0">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_eth0_conn">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Checking USB Port 1 (FRONT 2.0)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_eth1_conn">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>SYSTEM HEALTH CHECK</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Check List</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>LED STATUS</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label_usb_f3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="16" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_SW">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>SWITCH</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_CAM">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>CAMERA</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_SYS">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>SYSTEM</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_PWR">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>POWER</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="label_12">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>RESULT</string>
      </property>
     </widget>
    </item>
    <item row="17" column="1" colspan="2">
     <widget class="QLabel" name="label_13">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>NG</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
